packages/@aws-cdk/aws-events-targets/test/api-destination/api-destination.test.ts

```diff
@@ -0,0 +1,113 @@
+import { expect, haveResource, haveResourceLike } from '@aws-cdk/assert-internal';
+import * as events from '@aws-cdk/aws-events';
+import * as iam from '@aws-cdk/aws-iam';
+import { Duration, Stack } from '@aws-cdk/core';
+import * as targets from '../../lib';
+
+test('use api destination as an eventrule target', () => {
+  // GIVEN
+  const stack = new Stack();
+
+  const connection = new events.Connection(stack, 'Connection', {
+    authorizationType: events.AuthorizationType.BASIC,
+    authParameters: {
+      basicAuthParameters: {
+        password: 'password',
+        username: 'username',
+      },
+    },
+    description: 'ConnectionDescription',
+    connectionName: 'testConnection',
+  });
+
+  const destination = new events.ApiDestination(stack, 'Destination', {
+    apiDestinationName: 'apiDestinationName',
+    connection: connection,
+    invocationEndpoint: 'https://endpoint.com',
+    invocationRateLimit: Duration.seconds(10),
```

Comment:
@msimpsonnz  `invocationRateLimit` should be the `The maximum number of requests per second to send to the HTTP invocation endpoint.`. 

Why in that test you are setting `Duration.seconds(10)` instead of `10` ?